api-client/src/utils/generateUID.ts

```diff
@@ -0,0 +1,10 @@
+const DEFAULT_LENGTH = 20
+const generateUID = (length = DEFAULT_LENGTH): string => {
+  let generatedID = ''
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
+  for (let i = 0; i < length; i += 1) {
+    generatedID += characters.charAt(Math.floor(Math.random() * characters.length))
+  }
+  return `${generatedID}`
+}
+export default generateUID
```

Comment:
Did somebody check with the CMS team if they can send the UID like Steffen suggested in https://issues.tuerantuer.org/browse/IGAPP-1152 ?

Reply:
i addressed this to cms in mattermost. But this is not a blocker for this ticket. thx for mention it